test/opt/pass_utils.h

```diff
@@ -49,6 +49,16 @@ std::string JoinAllInsts(const std::vector<const char*>& insts);
 // will be ignored.
 std::string JoinNonDebugInsts(const std::vector<const char*>& insts);
 
+// Returns a vector that contains the contents of |a| followed by the contents
+// of |b|.
+template <typename T>
+std::vector<T> Concat(const std::vector<T>& a, const std::vector<T>& b) {
```

Comment:
This is fine.
But many uses of this function is actually really concatenating 3 vectors.
Elsewhere we've made a function that takes a vector of vectors.  That involves more copies, so it isn't great.  Maybe just add a 2nd overload that takes 3 arguments. ?

Definitely optional, though.